overlord/devicestate/devicemgr.go

```diff
@@ -2214,3 +2214,123 @@ func (m *DeviceManager) RemoveRecoveryKeys() error {
 
 	return secbootRemoveRecoveryKeys(recoveryKeyDevices)
 }
+
+// EncryptionRequirements
+// TODO: move to secboot/encryption.go (?)
```

Comment:
maybe yes, if we can move the function too though?

Reply:
Sadly we won't be able to use the function as we need to query the fde hooks which is only available in DeviceManager. I removed this XXX now as it's resolved.